### ISSUE TITLE
Issue with Keras training

### ISSUE THREAD

**Issue by li8bot** [Tessellate-Imaging/monk_v1#5]
Validation loss improves but validation accuracy does not.

**Comment by abhi-kumar** [comment]
High learning rates are leading models to not converge. Thus the initial learning rate in default modes are as low as 0.0001 and 0.001.

Switched plateau scheduler to step scheduler with a multiplication factor of 0.97.
Epoch 1 - 0.0001
Epoch 2 - 0.0001*0.97
Epoch 3 - 0.0001*0.97*0.97
... and so on 

Comment if validation loss decreases and accuracy improves or the problem still persists.

**Comment by abhi-kumar** [comment]
Closing the issue. Reopen if error still persists.
